protocol/messenger_handler.go

```diff
@@ -869,6 +877,29 @@ func (m *Messenger) HandleContactUpdate(state *ReceivedMessageState, message pro
 
 	logger.Info("Handling contact update")
 
+	if message.ContactRequestPropagatedState != nil {
+		result := contact.ContactRequestPropagatedStateReceived(message.ContactRequestPropagatedState)
+		if result.sendBackState {
+			// This is a bit dangerous, since it might trigger a ping-pong of contact updates
```

Comment:
How about we track this problem as an issue and remove the comment? It seems rather important.

Reply:
I am not sure how to address this, but I can add an issue to at least have a look to make sure that we don't create a storm of contact requests :)

Reply:
I kind of like your idea of using an exponential backoff to decide if/when to send back the state. It sounds reasonable and not too difficult to implement, but maybe it could cause side-effects I'm not aware of.

Reply:
I think that's useful to avoid spamming and we will implement it, but might hide the issue at this stage, say that the two states never converge, and keep messaging each other, then it would be hard to see from logs, while now it would be easier since we would see a lot of requests being exchanged, though 100% something we want to implement

Reply:
You're right, the matter deserves better analysis.